run_oscar.json

```diff
@@ -0,0 +1,107 @@
+{
+    "id": "run_oscar",
+    "summary": "Run an external OSCAR service.",
+    "description": "Run an external docker-based OSCAR service.",
+    "categories": [
+        "oscar",
+        "service"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "token_env_var",
+            "description": "OpenEO auth token environment variable.",
+            "schema": {
+                "title": "token-env-var",
+                "type": "string",
+                "subtype": "string",
+                "pattern": "^[a-zA-Z0-9_\\-\\.]+$"
+            }
+        },
+        {
+            "name": "oscar_endpoint",
+            "description": "The URL of the OSCAR service.",
+            "schema": {
+                "title": "oscar-url",
+                "type": "string",
+                "format": "url",
+                "subtype": "url",
+                "pattern": "^https?://"
+            }
+        },
+        {
+            "name": "service",
+            "description": "The OSCAR service to be executed.",
+            "schema": {
+                "title": "service",
+                "type": "string",
+                "subtype": "string"
+            }
+        },
+        {
+            "name": "output",
```

Comment:
The result should be re-usable by other openEO processes. If the result is provided as STAC Collection, in my opinion the output should be a "datacube" type, and then handled in the back-end how to pass it further. Edit: We don't need this parameter, it's defined in the "returns" part, remove it.